openaerostruct/geometry/tests/test_geometry_mesh_transformations.py

```diff
@@ -132,6 +138,28 @@ def test_scalex_symmetry(self):
         check = prob.check_partials(compact_print=True, abs_err_tol=1e-5, rel_err_tol=1e-5)
         assert_check_partials(check, atol=1e-6, rtol=1e-6)
 
+    def test_scalex_chord_scaling_pos(self):
+        symmetry = False
+        mesh = get_mesh(symmetry)
+
+        prob = om.Problem()
+        group = prob.model
+
+        val = self.rng.random(NY)
+        chord_scaling_pos = self.rng.random(1)
```

Comment:
Can you pick a value here, maybe 0 or 1, that you can analytically check? Then add an assert after run model that checks that the chord has been modified as you expect

Reply:
Is this still an open item?